README.md

```diff
@@ -228,20 +228,111 @@ Now everything has been set up and OP-TEE is ready to be used.
-#### 4.3.1 Get the compiler and source
-Follow the instructions in the "4.1 Basic setup".
 
-#### 4.3.2 Build
+WARNING: 
+setup_juno_optee.sh script provides a coherent set of components (OP-TEE client/driver/os,
+Linux kernel version 3-16.0-rc5)
+Further release will align the ARM Juno setup with other OP-TEE supported platforms:
++ Linux kernel version alignment (3.18-rc1) with QEMU/FVP (DMA_BUF API change)
++ Will need arch/arm/Kconfig patch(es) (i.e DMA_SHARED_BUFFER etc...)
+Temporary patch files required for linux kernel and juno dtb definition.
+config.linux-linaro-tracking.a226b22057c22b433caafc58eeae6e9b13ac6c8d.patch
+juno.dts.linux-linaro-tracking.a226b22057c22b433caafc58eeae6e9b13ac6c8d.patch
+
+#### 4.3.1 Prerequisites 
++ The following packages must be installed:
+
 ```
-$ cd optee_os
-$ PLATFORM=vexpress PLATFORM_FLAVOR=juno CROSS_COMPILE=arm-linux-gnueabihf- make
+$ sudo apt-get install zlib1g-dev libglib2.0-dev libpixman-1-dev libfdt-dev \
+		       libc6:i386 libstdc++6:i386 libz1:i386 cscope
 ```
 
-#### 4.3.3 Prepare the images to run on Juno
-Will be written soon.
++ ARM Juno pre-built binary(ies)
+ARM Juno Pre-built binary bl30.bin (SCP runtime)
+ARM Juno Pre-built binary bl33.bin (UEFI)
+Download at http://community.arm.com/docs/DOC-8401
 
-#### 4.3.4 Boot and run the software on Juno
-Will be written soon.
+
+#### 4.3.2 Download and install ARM Juno
+```
+$ wget https://raw.githubusercontent.com/OP-TEE/optee_os/master/scripts/setup_juno_optee.sh
+$ chmod 711 setup_juno_optee.sh
+$ ./setup_juno_optee.sh
+```
+
+#### 4.3.3 Build
++ List of helper scripts generated during installation:
+
+* `build_atf_opteed.sh`: This is used to build ARM-Trusted-Firmware and must be
+  called when you have updated any component that are included in the FIP (like
+  for example OP-TEE os).
+
+* `build_linux.sh`: This is used to build the Linux Kernel.
+
+* `build_normal.sh`: This is a pure helper script that build all the normal
+   world components (in correct order).
+
+* `build_optee_client.sh`: This will build OP-TEEs client library.
+
+* `build_optee_linuxdriver.sh`: This will build OP-TEEs Linux Kernel driver (as
+   a module).
+
+* `build_optee_os.sh`: Builds the Trusted OS itself.
+
+* `build_optee_tests.sh`: This will build the test suite (pay attention to the
+   access needed).
+
+* `build_secure.sh`: This is the helper script for the secure side that will
+  build all secure side components in the correct order.
+
+* `clean_gits.sh`: This will clean all gits. Beware that it will not reset the
+  commit to the one used when first cloning. Also note that it will only clean
+  git's.
+
++ Run the scripts in the following order:
+```
+$ ./build_secure.sh
+$ ./build_normal.sh
+```
+
+#### 4.3.4 Booting Up ARM Juno
++ Update the ARM Juno embedded flash memory:
+path: JUNO/SOFTWARE
+files:bl1.bin, fip.bin, Image, juno.dtb
++ Copy OP-TEE binaries (client libraries, supplicant, driver, CAs, TAs) on the USB key (filesystem(*))
```

Comment:
I'd prefer having a bullet list naming exactly all needed binaries, to rule out any eventual question.